docker-compose.yml

```diff
@@ -17,5 +17,3 @@ services:
     tty: true
     depends_on:
       - db
-    volumes:
-      - ./dbschema:/dbschema
```

Comment:
This was sort of useful for local development, but since we also use this dockerfile in `virtual-coach-main` I removed it.

Reply:
Yes this seems fair enough. It makes sense to converge on "one way to do it" to simplify dev/deploy.